### PR TITLE
MEPTS-471 - Fix Tx-TB Denominator (Positive Result Returned)

### PR DESCRIPTION
**Pull request by iteliolucas**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java [hunk 1]
```diff
@@ -540,7 +540,7 @@ public static String getPatientsWithObsBetweenDates(
     s.append("WHERE e.location_id = :location AND e.encounter_type = ${encounterType} ");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java:540]: @iteliolucas can you please change this names for sugestive names like like the proper name of encounter something like this, is difficult to see if this query is ohk or not like this 
```suggestion
    s.append("WHERE e.location_id = :location AND e.encounter_type = ${adultoSeguimentoEncounterType} ");
```

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java [hunk 2]
```diff
@@ -540,7 +540,7 @@ public static String getPatientsWithObsBetweenDates(
...
     s.append("AND o.concept_id = ${question}  ");
     s.append("AND o.value_coded in (${answers}) ");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java:542]: Same here

```suggestion
    s.append("AND o.concept_id = ${malariaConcept}  ");
    s.append("AND o.value_coded in (${aspirinaConcept}) ");
```

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java [hunk 3]
```diff
@@ -302,42 +302,6 @@ public CohortDefinition artList() {
...
-
-    cd.addSearch("test-tb-lam", EptsReportUtils.map(getTestTBLAM(), mappings));
-
-    cd.addSearch("culture-test", EptsReportUtils.map(getCultureTest(), mappings));
-
-    cd.setCompositionString(
-        "genexpert-positivo OR result-basiloscopia OR test-tb-lam OR culture-test");
-
-    return cd;
-  }
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java:340]: @iteliolucas Any reason to remove this?

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java (hunk 1 above)
Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java:540]: I guess this is like this cause is generic one, never mind the sugestion.....
Please move this to genericQueries

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java (hunk 2 above)
Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java:542]: I guess this is like this cause is generic one, never mind the sugestion.....
Please move this to genericQueries

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java (hunk 3 above)
Comment (iteliolucas) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java:340]: This piece of code is no longer in use, and should have been removed in the latest fix.

**Pull request merged**